specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.1/backuprestore.json

```diff
@@ -82,6 +82,62 @@
         }
       }
     },
+    "/prebackup": {
+      "post": {
+        "tags": [
+          "PreFullBackup"
+        ],
+        "operationId": "PreFullBackup",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "description": "Pre-backup operation for checking whether the customer can perform a full backup operation.",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "preBackupOperationParameters",
+            "schema": {
+              "$ref": "#/definitions/PreBackupOperationParameters"
+            },
+            "description": "Optional parameters to validate prior to performing a full backup operation."
+          },
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
```

Comment:
Can there be a sync response as well here, if yes does it need to be defined seperately?

Reply:
No Chandrasekaran. We are supporting only async.